docs/writing-rules/rule-syntax.md

```diff
@@ -32,7 +32,7 @@ This document describes Semgrep’s YAML rule syntax.
 | :--------- | :------- | :---------------------------------- |
 | [`options`](#options)   | `object` | Options object to enable/disable certain matching features |
 | [`fix`](#fix)           | `object` | Simple search-and-replace autofix functionality  |
-| [`metadata`](#metadata) | `object` | Arbitrary user-provided data; attach data to rules without affecting Semgrep’s behavior |
+| [`metadata`](#metadata) | `object` | User-provided data; attach data to rules without affecting Semgrep behavior |
```

Comment:
```suggestion
| [`metadata`](#metadata) | `object` | Arbitrary user-provided data; attach data to rules without affecting Semgrep behavior |
```

It's arbitrary because you can enter any key under metadata and Semgrep will accept it, unlike other keys, which are specific.